DEVELOPMENT.md

```diff
@@ -160,40 +166,67 @@ $ cd showcase
 $ mvn compile -P update
 ```
 
-## Running the Plugin
+## Running the Plugin under googleapis with local gapic-generator-java
 
 See also above section "Showcase Integration Testing".
 
 To generate a production GAPIC API:
 
 1. Clone [googleapis](https://github.com/googleapis/googleapis).
 
-2. Point to local gapic-generator-java
+2. Modify `googleapis/WORKSPACE` to point to local gapic-generator-java
 
-   Normally, googleapis's build pulls in googleapis/gapic-generator-java from the
-   Internet:
+   Normally, googleapis's build pulls in gapic-generator-java from Maven Central. 
+   For a local run, we first need to build a local SNAPSHOT jar of the generator. Then we point googleapis to 
+   both the local SNAPSHOT jar and the local copy of the generator.
 
+   Before:
    ```
-   # Java microgenerator.
-   …
-   _gapic_generator_java_version = "2.1.0"
-   
-   http_archive(
-       name = "gapic_generator_java",
-       …
-       urls = ["https://github.com/googleapis/gapic-generator-java/archive/v%s.zip" % _gapic_generator_java_version],
-   )
+    _gapic_generator_java_version = "2.13.0"
+
+    maven_install(
+        artifacts = [
+            "com.google.api:gapic-generator-java:" + _gapic_generator_java_version,
+        ],
+        #Update this False for local development
+        fail_on_missing_checksum = True,
+        repositories = [
+            "m2Local",
+            "https://repo.maven.apache.org/maven2/",
+        ]
+    )
+    
+    http_archive(
+        name = "gapic_generator_java",
+        strip_prefix = "gapic-generator-java-%s" % _gapic_generator_java_version,
+        urls = ["https://github.com/googleapis/gapic-generator-java/archive/v%s.zip" % _gapic_generator_java_version],
+    )
    ```
 
-   By replacing this portion using the built-in local_repository rule, you can make
-   it refer to your local development repo:
+   After
 
    ```
+    _gapic_generator_java_version = "2.13.1-SNAPSHOT"
+
+    maven_install(
+        artifacts = [
+           "com.google.api:gapic-generator-java:" + _gapic_generator_java_version,
+        ],
+        #Update this False for local development
+        fail_on_missing_checksum = False,
```

Comment:
(I'm Bazel novice) Can't this True/False be a variable, based on _gapic-generator_java_version has SNAPSHOT?

Reply:
I think it's doable as Bazel scripts are basically a Python variant, but we may want to test non-SNAPSHOT version locally sometimes.